spec/models/user_spec.rb

```diff
@@ -923,6 +923,14 @@
           user.suspend!
         end
 
+        it 'send account suspension push event' do
+          expect(PushNotification::HttpPush).to receive(:deliver).once.
+            with(PushNotification::AccountSuspendedEvent.new(
+              user: user,
+            )).ordered
```

Comment:
what does `.ordered` do for us here?

Reply:
Oh, there's no need to order it, as we only have one notification, and I copied this from another location but missed it.